src/main/java/de/zalando/aruha/nakadi/domain/Subscription.java

```diff
@@ -0,0 +1,88 @@
+package de.zalando.aruha.nakadi.domain;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Collections;
+import java.util.Set;
+
+import static com.google.common.collect.Sets.newTreeSet;
+
+public class Subscription {
+
+    @Nullable
+    private String id;
+
+    @NotNull
+    private String owningApplication;
+
+    @NotNull
+    @Size(min = 1, max = 1)
+    private Set<String> eventTypes;
+
+    private String useCase = "none";
+
+    private DateTime createdAt = new DateTime(DateTimeZone.UTC);
```

Comment:
I was thinking if should it be done here or in the persistence layer. Managing this in the persistence layer seems to make sense because this piece of information is related to persistence. I could even imagine a generic class handling these updates for every database update for every table.